R/install.R

```diff
@@ -286,6 +287,16 @@ check_install_dir <- function(dir_cmdstan, overwrite = FALSE) {
   TRUE
 }
 
+github_auth_token <- function() {
+  github_pat <- Sys.getenv("GITHUB_PAT")
+  if (nzchar(github_pat)) {
+    auth_token <- c(Authorization = paste0("token ", github_pat))
```

Comment:
With the auth token from Github we are not limited with Github API requests anymore. Users can also set this locally, though I cant see anyone needing it really.